### ISSUE TITLE
latest 3.5 version on xenial?

### ISSUE THREAD

**Issue by step21** [deadsnakes/issues#55]
### description

Repo added, seems to pull updates, but upon `apt install python3.5` it reports that the newest (3.5.2) is already installed, whereas I hoped to install 3.5.4 which should be in the ppa, right?

### os information
Ubuntu 16.04.3
#### lsb_release -a

```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.3 LTS
Release:	16.04
Codename:	xenial
```

#### uname -a

```
Linux hostname 4.4.0-109-generic #132-Ubuntu SMP Tue Jan 9 19:52:39 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by asottile** [comment]
Since `python3.5` is part of ubuntu for xenial we won't have a package for it (we also don't have a 3.4 package for trusty).  This is for a couple reasons:
- Updating from one patch version to another generally _isn't_ a safe operation
- Since python is tightly coupled with lots of system packages it's not generally a good idea to use a non-upstream package for it (this is the same reason we've discontinued the `deadsnakes-python2.7` repository).

That said, if you would like to footgun this, you _could_ try using the `trusty` package on xenial.  I strongly do not recommend this for the above reasons -- though it might work.

We do provide [`3.5.4` for trusty](https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa/+packages?field.name_filter=3.5&field.status_filter=published&field.series_filter=) however

**Comment by step21** [comment]
:-/ Well personally I wouldn't care, but apparently some asyncio stuff is broken in 3.5.1/2 and fixed in 3.5.4 I guess it is easier to try 3.6 than using a trusty package. Thanks for the reply.

**Comment by asottile** [comment]
Ah yeah you're probably better off using 3.6 :)

Thanks for the issue!
